content/implementing-user-authentication-and-authorization-in-a-loopback-rest-api/index.md

```diff
@@ -0,0 +1,542 @@
+---
+title: "Implementing User Authentication and Authorization in a Loopback REST API"
+date: "2021-12-17"
+coverImage: "coverImage.jpg"
```

Comment:
Please change `coverImage.jpg` to something relevant to the topic of Loopback. For example: https://www.loginradius.com/blog/async/guest-post/securing-flask-api-with-jwt/